src/Monolog/Formatter/WildfireFormatter.php

```diff
@@ -90,7 +91,7 @@ public function format(array $record): string
         // The message itself is a serialization of the above JSON object + it's length
         return sprintf(
             '%d|%s|',
-            strlen($json),
+            Utils::strlen($json),
```

Comment:
I'd guess this needs to be a real strlen at binary level and not counting unicode characters, but not 100% sure